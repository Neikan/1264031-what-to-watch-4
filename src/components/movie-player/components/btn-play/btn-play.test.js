// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";

// Импорт компонентов
import BtnPlay from "./btn-play.jsx";


describe(`Test BtnPlay component`, () => {
  test(`BtnPlay component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <BtnPlay />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
