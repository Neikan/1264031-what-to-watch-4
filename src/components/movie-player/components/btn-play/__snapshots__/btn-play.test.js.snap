// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test BtnPlay component BtnPlay component is created and rendered correctly 1`] = `
Array [
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>,
  <span>
    Play
  </span>,
]
`;
